misc/package/Data Files/MWSE/core/lib/DependencyManager/v1/init.lua

```diff
@@ -296,53 +297,72 @@ function DependencyManager:doLuaModCheck(dependency)
             return self:doVersionCheck(dependency, init.version)
         end
     end
-
     return false, string.format("Could not find dependency: %s", dependency.luaMod)
 end
 
+function DependencyManager:checkBuildDate()
+    if not self.metadata.buildDate then return true end
+    if mwse.buildDate == nil  then
+        self.logger:error("Could not get current build date")
+        return false
+    end
+    return mwse.buildDate <= self.metadata.buildDate
+end
+
 ---@return boolean #returns true if all dependencies passed, false if any failed.
 function DependencyManager:checkDependencies()
     local failedDependencies = {} ---@type table<string, DependencyManager.failedDependency>
-    local didFail = false
-    for _, dependency in pairs(self.dependencies) do
-        self.logger:debug("Checking dependency: %s", dependency.name)
-        if dependency.plugin then
-            self.logger:debug("Checking plugin dependency: %s", dependency.plugin)
-            if not tes3.isModActive(dependency.plugin) then
-                failedDependencies[dependency.name] = failedDependencies[dependency.name] or {
-                    dependency = dependency,
-                    reasons = {}
-                }
-                table.insert(failedDependencies[dependency.name].reasons,
-                    string.format('Plugin "%s" is not active', dependency.plugin))
-                didFail = true
+    if not self:checkBuildDate() then
+        failedDependencies["MWSE"] = {
+            dependency = {
+                name = "MWSE",
+            },
+            reasons = {
+                string.format("Build date is too old. Current build date: %s", self.metadata.buildDate)
```

Comment:
`string.format("Build date is too old. Current build date: %s. Required build date: %s", mwse.buildDate, self.metadata.buildDate)`?